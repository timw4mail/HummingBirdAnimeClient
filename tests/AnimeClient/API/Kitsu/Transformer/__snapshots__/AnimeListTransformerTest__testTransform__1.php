<?php return Aviat\AnimeClient\Types\AnimeListItem::__set_state(array(
   'id' => '15839442',
   'mal_id' => '33206',
   'episodes' => 
  array (
    'watched' => '-',
    'total' => '-',
    'length' => NULL,
  ),
   'airing' => 
  array (
    'status' => 'Currently Airing',
    'started' => '2017-01-12',
    'ended' => NULL,
  ),
   'anime' => 
  Aviat\AnimeClient\Types\Anime::__set_state(array(
     'age_rating' => NULL,
     'cover_image' => 'https://media.kitsu.io/anime/poster_images/12243/small.jpg?1481144116',
     'genres' => 
    array (
      0 => 'Comedy',
      1 => 'Fantasy',
      2 => 'Slice of Life',
    ),
     'id' => '12243',
     'show_type' => 'TV',
     'slug' => 'kobayashi-san-chi-no-maid-dragon',
     'streaming_links' => 
    array (
    ),
     'title' => 'Kobayashi-san Chi no Maid Dragon',
     'titles' => 
    array (
      0 => 'Miss Kobayashi\'s Dragon Maid',
      1 => '小林さんちのメイドラゴン',
    ),
  )),
   'notes' => NULL,
   'private' => false,
   'rewatching' => false,
   'rewatched' => 0,
   'user_rating' => '-',
   'watching_status' => 'current',
));
