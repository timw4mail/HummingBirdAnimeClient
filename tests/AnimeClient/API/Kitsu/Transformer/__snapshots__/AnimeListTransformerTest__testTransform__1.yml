empty: false
id: '435513'
mal_id: '209'
episodes:
    watched: 7
    total: 26
    length: 1500
airing:
    status: 'Finished Airing'
    started: '2003-09-01'
    ended: '2004-03-16'
anime:
    empty: false
    age_rating: PG
    cover_image: 'https://media.kitsu.io/anime/poster_images/185/small.jpg?1597697520'
    genres: {  }
    id: '185'
    show_type: TV
    slug: r-o-d
    streaming_links: [{ meta: { name: Crunchyroll, link: true, image: streaming-logos/crunchyroll.svg }, link: 'http://www.crunchyroll.com/rod', subs: [en], dubs: [ja] }, { meta: { name: Hulu, link: true, image: streaming-logos/hulu.svg }, link: 'http://www.hulu.com/rod-the-tv', subs: [en], dubs: [ja] }]
    title: 'R.O.D the TV'
    titles: ['アール・オー・ディー ザ・ティーヴィー']
notes: ''
private: false
rewatching: false
rewatched: 0
user_rating: '-'
watching_status: CURRENT
